package.lisp

```diff
@@ -21,6 +23,13 @@ All ARGS are declared as `ignorable'."
                                              lambda-list-keywords)))
        ,@body)))
 
+(defun slot-names (class-specifier)
+  ;; TODO: `slot-names' or `direct-slot-names'?
+  #-ecl
+  (mopu:slot-names class-specifier)
+  #+ecl
```

Comment:
Actually, why? Seems like `mopu:slot-names` definition is exactly what you're reproducing here:
```
(defun class-slot-names (thing)
  (let ((class (get-class thing)))
    (if class
      (mapcar 'mop:slot-definition-name
	      (mop:class-slots (finalize-class-if-necessary class)))
      (progn
	(warn "class for ~a not found)" thing)
	nil))))
```